benchmark/benchmark17_test.go

```diff
@@ -0,0 +1,64 @@
+// +build go1.7
+
+package benchmark
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/benchmark/stats"
+)
+
+func BenchClient(b *testing.B, maxConcurrentCalls, reqSize, respSize int, unaryMode, enableTrace bool) {
+	if enableTrace {
+		grpc.EnableTracing = true
+	} else {
+		grpc.EnableTracing = false
+	}
+	if unaryMode {
```

Comment:
We probably should keep Unary and Streaming separate, as two sets of benchmark tests, because we may want to send multiple requests in the streaming call. But we can keep this in this PR, and change it later when we need.